### ISSUE TITLE
AndroidAuto rendering lags and stepping

### ISSUE THREAD

**Issue by garminbreak** [organicmaps/organicmaps#9886]
AndroidAuto rendering lags and stepping in drive, there is small delay between reality and rendered position, in 50 km per hour approx. 10-15 meters in places with perfect GPS receipt.

**Steps to reproduce**
1. Connect Phone to car multimedia unit
2. Activate AndroidAuto
3. Start OragnicMaps as navigation application
4. Drive and observe, no matter with active or without active navigation

**Expected behaviour**
Fluid flow animation

**Video**
Video captured in one of cars available to me, but result is same in all tested cars 
https://drive.google.com/file/d/1x9NqsIWKwsueGP3b7Uy1H_XmiJr05bsW/view?usp=sharing
 

**System information:**
 - many AndroidAuto enabled cars
 - Organic Maps version: 2024.11.27-12-Google
 - Device Model: phone Samsung Galaxy A52s 5G, Samsung Galaxy A55 5G

**Additional context**
Application Maps.ME behaves very similar

**Comment by biodranik** [comment]
Are you talking about a jumping position in a "free-drive" mode? The position arrow updates with each GPS update. The smooth position animation into a _predicted_ location (like in navigation mode) has not been implemented yet.

**Comment by garminbreak** [comment]
Yes, it is. But in my case position jumps even with active navigation, not only in free-drive.

**Comment by biodranik** [comment]
Navigation mode should be smooth, strange. Does it also jump if you use the navigation on a phone without connecting it to AA?

Can you please enable logs in OM settings, and ideally also record the screen of jumping position in the nav mode? Then send logs with a reference to this issue by pressing "Report a bug" button.

**Comment by garminbreak** [comment]
OK, here is video
https://github.com/user-attachments/assets/93f8bacc-acbe-410a-b13c-465ccbb005e1

As You can see, position is jumping.

**Comment by silmarine** [comment]
I am also having this happen on Android Auto. My phone is a Pixel 7a with stock ROM, if that's helpful at all.

**Comment by ekirchman** [comment]
I've noticed the same issues as well on my Pixel 5a with Graphene OS. I may be wrong, but was the location predictor removed for Android? From the looks of this commit, it appears to be taken out of the codebase a while ago: https://github.com/organicmaps/organicmaps/commit/5c4967ae89dc328eca5dfe932d32fe81af9147dc .

**Comment by emabee1** [comment]
i have the same issue on a pixel 7
